gateway/src/api/latest.rs

```diff
@@ -1161,24 +768,13 @@ impl ApiBuilder {
 
     pub fn into_router(self) -> Router {
         let service = self.service.expect("a GatewayService is required");
-        let sender = self.sender.expect("a task Sender is required");
         let auth_cache = self.auth_cache.expect("an auth cache is required");
         let auth_client = self.auth_client.expect("an auth client is required");
 
-        // Allow about 4 cores per build
-        let mut concurrent_builds = num_cpus::get() / 4;
```

Comment:
If `num_cpus` was a crate, can we also remove it?

Reply:
Good catch!